src/Models/BaseElement.php

```diff
@@ -295,6 +295,10 @@ public function getCMSFields()
                     )
             );
 
+            // Rename the "Settings" tab
+            $fields->fieldByName('Root.Settings')
+                ->setTitle(_t(__CLASS__ . '.SettingsTabLabel', 'Content'));
```

Comment:
You're renaming "Settings" to "Content"? Do you think that might be a little misleading?

Reply:
oops, copy/paste error, thanks for the heads up!